src/planet-4-151612/openresty/etc/GeoIP.conf

```diff
@@ -1,11 +1,16 @@
-# The following AccountID and LicenseKey are required placeholders.
-# For geoipupdate versions earlier than 2.5.0, use UserId here instead of AccountID.
-# AccountID 0
-UserId 0
-LicenseKey 000000000000
+# GeoIP.conf file for `geoipupdate` program, for versions >= 3.1.1.
+# Used to update GeoIP databases from https://www.maxmind.com.
+# For more information about this config file, visit the docs at
+# https://dev.maxmind.com/geoip/geoipupdate/.
+# Account is currently registered with jhasenau@greenpeace.org
 
-# Include one or more of the following edition IDs:
-# * GeoLite2-City - GeoLite 2 City
-# * GeoLite2-Country - GeoLite2 Country
-# For geoipupdate versions earlier than 2.5.0, use ProductIds here instead of EditionIDs.
-ProductIds GeoLite2-City GeoLite2-Country
+# `AccountID` is from your MaxMind account.
+AccountID 146520
```

Comment:
Is hard-coding these secrets in publically accessible VCS a good idea?

These configuration files should likely be generated with dockerize, and the secrets injected by env vars on startup.

Will require modifying Helm charts (planet4 and static), CI environment, etc.